mutagen/asf/_objects.py

```diff
@@ -381,6 +381,8 @@ def parse(self, asf, data):
             obj.parse(asf, data[22 + datapos + 24:22 + datapos + size])
             self.objects.append(obj)
             datapos += size
+            if size < 1:
```

Comment:
shouldn't this throw on anything < 24?

Reply:
Isn't it possible to have several objects overlapping each other?